lib/utils/featureDetection.ts

```diff
@@ -1,7 +1,34 @@
+import { request as nodeHttpRequest } from "http";
+import { request as nodeHttpsRequest } from "https";
```

Comment:
I think this will cause a polyfill for the node default modules in the regular bundle, unless you specifically exclude them in rollup

Reply:
looks like you are doing that